OxfordBodleian/Juel-Jensen/BDLaethb3.xml

```diff
@@ -33,15 +33,583 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                             ወኵሉ፡ ቦቱ፡ ኮነ<hi rend="rubric" rendition="#partialRubric">፤</hi> ወዘእንበሌሁሰ፡ አልቦ፡ ዘኮነ፡
+                                         </incipit>
+                                         <explicit xml:lang="gez">
+                                             <locus target="#293va"/>
+                                             ወሶበ፡ ተጽሕፈ፡ ኵሉ፡ <add place="overstrike">በበ፡</add>  አሐዱ፡ እምኢያግመሮ፡ ዓለም፡ ጥቀ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                             መጻሕፍቲሁ፡ ዘተጽሕፈ<hi rend="rubric" rendition="#partialRubric">፨</hi> አሜን<hi rend="rubric" rendition="#partialRubric">፨</hi> ወአሜን<hi rend="rubric" rendition="#partialRubric">፨</hi>                                         </explicit>
+                                     </msItem>
+                                     
+                                     <msItem xml:id="ms_i1.5.2">
+                                         <locus target="#293vb"/>
+                                         <title type="complete" ref="LIT1693John#Postscript"/>
+                                         <incipit xml:lang="gez">
+                                             <locus target="#293vb"/>
+                                             <hi rend="rubric">መልአ፡</hi> <seg type="title subscriptio"><hi rend="rubric">ስብከተ፡ ዮሐንስ፡ ወልደ፡ ዘብዴዎስ፡</hi> ሐዋርያ፡ 
+                                                 እምዓሠርቱ፡ ወክልኤቱ<hi rend="rubric" rendition="#partialRubric">፨</hi></seg> እንተ፡ ጸሐፋ፡ በዮናኒ፡ ለሰብአ፡ ሀገረ፡ ኤፌሶን።
+                                         </incipit>
+                                         <explicit xml:lang="gez">
+                                             <locus target="#293vb"/>
+                                             በሰመንቱ፡ ዓመተ፡ መንግሥቱ፡ ለኔሮን፡ ቄሳር<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                             ወስብሐት፡ ለእግዚአብሔር፡ 
+                                             ለዓለመ፡ ዓለም<hi rend="rubric" rendition="#partialRubric">፨</hi> ወትረ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                             አሜን<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                         </explicit>
+                                     </msItem>
+                                 </msItem>
+                                 
+                             </msItem>
+                             </msItem>
+                    </msContents>
+                    
 
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p><locus target="#14"/> is cut off after the first column. Very rare parchment repairs (e.g. <locus target="#202"/>), presumably from the 
+                                        parchmenting process.
```

Comment:
I think that in the end we will have to check all the formulas used to indicate the process. At the moment they vary from parchmenting process, parchment making, and parchment making process. For me they are all synonyms but perhaps among these there is a more formal one.

Reply:
Absolutely, we need to do this category by category

Reply:
Thanks, if I understood this line of conversation, I don't think we need to standardize everything we say: I think it is fine to use synonyms.